server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -463,6 +464,7 @@ export class MongoDbFactory implements core.IDbFactory {
 	private readonly globalDbEndpoint?: string;
 	private readonly connectionPoolMinSize?: number;
 	private readonly connectionPoolMaxSize?: number;
+	private readonly directConnection?: boolean;
```

Comment:
This don't need to be optional, please keep above optional though

Reply:
Done.